ambry-account/src/test/java/com/github/ambry/account/AccountTestUtils.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.account;
+
+import com.github.ambry.utils.Utils;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Random;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.junit.Assert.*;
+
+
+/**
+ * Utils for testing account-related classes.
+ */
+class AccountTestUtils {
+  private static final Random random = new Random();
+
+  /**
+   * Assert a collection of {@link Account}s exist in the {@link AccountService}.
+   * @param accounts The collection of {@link Account}s to assert their existence.
+   * @param expectedAccountCount The expected number of {@link Account}s in the {@link AccountService}.
+   * @param accountService The {@link AccountService} to assert {@link Account}s existence.
+   */
+  static void assertAccountsInAccountService(Collection<Account> accounts, int expectedAccountCount,
+      AccountService accountService) {
+    Objects.requireNonNull(accounts, "accounts cannot be null");
+    Objects.requireNonNull(accountService, "accountService cannot be null");
+    assertEquals("Wrong number of accounts in accountService", expectedAccountCount,
+        accountService.getAllAccounts().size());
+    for (Account account : accounts) {
+      assertAccountInAccountService(account, accountService);
+    }
+  }
+
+  /**
+   * Asserts that an {@link Account} exists in the {@link AccountService}.
+   * @param account The {@link Account} to assert existence.
+   * @param accountService The {@link AccountService} to assert {@link Account} existence.
+   */
+  static void assertAccountInAccountService(Account account, AccountService accountService) {
+    Objects.requireNonNull(account, "account cannot be null");
+    Objects.requireNonNull(accountService, "accountService cannot be null");
+    Account accountFoundById = accountService.getAccountById(account.getId());
```

Comment:
don't we have Account equivalency and Container equivalency static methods ? I thought for testing the accountService, you might have been in need of one such method.